src/main/java/com/shopify/store/Application.java

```diff
@@ -0,0 +1,57 @@
+package com.shopify.store;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+/**
+ *  Application here where one can enter and add in product, we can also input an ENUM check in the future to look at categories of products
+ */
+public class Application {
+    public static BigDecimal cart (ArrayList<FruitProduct> cartList) {
```

Comment:
I think a `ShoppingCart` abstraction would be worthwhile, rather than using a raw arraylist directly.